pkg/tuned/controller.go

```diff
@@ -1010,15 +1014,26 @@ func (c *Controller) changeSyncerTuneD(change Change) (synced bool, err error) {
 		}
 		reload = reload || changeProvider
 
-		if (c.daemon.recommendedProfile != change.recommendedProfile) || change.nodeRestart {
+		inplaceUpdate = (c.daemon.recommendedProfile == change.recommendedProfile)
+		if !inplaceUpdate || change.nodeRestart {
 			if err = TunedRecommendFileWrite(change.recommendedProfile); err != nil {
 				return false, err
 			}
-			klog.V(1).Infof("recommended TuneD profile changed from %q to %q [deferred=%v nodeRestart=%v]", c.daemon.recommendedProfile, change.recommendedProfile, change.deferred, change.nodeRestart)
+
+			prevRecommended := c.daemon.recommendedProfile
 			// Cache the value written to tunedRecommendFile.
 			c.daemon.recommendedProfile = change.recommendedProfile
-			reload = true
-		} else if !change.deferred && (c.daemon.status&scDeferred != 0) {
+			klog.V(1).Infof("recommended TuneD profile changed from %q to %q [inplaceUpdate=%v nodeRestart=%v]", prevRecommended, change.recommendedProfile, inplaceUpdate, change.nodeRestart)
```

Comment:
It can indeed, seen multiple times in the logs. I believe that this is worthy information, and the presence of `inplaceUpdate` helps explaining the meaning. I however reworded "updated" into "changed".
If you disagree feel free to point out, I can change it again.